staging/src/k8s.io/kubectl/pkg/drain/drain.go

```diff
@@ -180,12 +180,10 @@ func (d *Helper) GetPodsForDeletion(nodeName string) (*podDeleteList, []error) {
 				break
 			}
 		}
-		if status.delete {
-			pods = append(pods, podDelete{
-				pod:    pod,
-				status: status,
-			})
-		}
+		pods = append(pods, podDelete{
```

Comment:
Yeah, I agree with this change, especially after looking at https://github.com/kubernetes/kubernetes/blob/2808d7154dc820c2718d92eb2f000ffc5840b5d1/staging/src/k8s.io/kubectl/pkg/drain/filters.go#L160-L165.

Can you please add a comment here, why this is done so, this will prevent eventual "optimization" like that from the PR you're reverting.

Reply:
OK. I'll try to do this.